pandas/tests/copy_view/test_interp_fillna.py

```diff
@@ -132,7 +132,7 @@ def test_interpolate_object_convert_copies():
     arr_a = get_array(df, "a")
     msg = "Can not interpolate with method=pad"
     with pytest.raises(ValueError, match=msg):
-        df.interpolate(method="pad", inplace=True, downcast="infer")
+        df.interpolate(method="pad", inplace=False, downcast="infer")
```

Comment:
From the original issue, I think we only wanted to disallow `inplace` for `resample.interpolate`?

Reply:
Pretty sure the test was failing 🤔 But I can check it again!

Reply:
You are right @mroeschke, it was a Find&Replace mistake. Revert and works.